### ISSUE TITLE
Question about the testing set used in experiments

### ISSUE THREAD

**Issue by willywsm1013** [justplus/turn-detection#1]
Hi! Thank you for the great work on this project!
Could you share which testing set was used in your experiments, so we can fairly compare our results?

**Comment by justplus** [comment]
```python
dataset = load_dataset("json", data_files="SFT.jsonl")["train"]
dataset = dataset.shuffle(seed=42)

test_size = 500
eval_size = 100
train_dataset = dataset.select(range(test_size, len(dataset)))
test_dataset = dataset.select(range(test_size))
eval_dataset = test_dataset.select(range(eval_size))
```

As shown in the code above, the test set comes from **SFT.jsonl**, and you also need to set **seed=42**.
